rpcs3/rpcs3qt/find_dialog.cpp

```diff
@@ -0,0 +1,49 @@
+#include "find_dialog.h"
+
+#include <QVBoxLayout>
+
+find_dialog::find_dialog(QTextEdit* edit, QWidget *parent, Qt::WindowFlags f) : QDialog(parent, f), m_text_edit(edit)
+{
+	setWindowTitle(tr("Find string"));
+	setWindowFlags(windowFlags() & ~Qt::WindowContextHelpButtonHint);
+
+	m_find_bar = new QLineEdit();
+	m_find_bar->setPlaceholderText(tr("Search..."));
+
+	m_find_next = new QPushButton(tr("Next"));
+	m_find_last = new QPushButton(tr("Last"));
+
+	QHBoxLayout* layout = new QHBoxLayout();
+	layout->addWidget(m_find_bar);
+	layout->addWidget(m_find_next);
+	layout->addWidget(m_find_last);
+	setLayout(layout);
+
+	m_find_bar->setFixedWidth(sizeHint().width() * 0.7f);
+
+	connect(m_find_next, &QPushButton::pressed, this, &find_dialog::find_next);
+	connect(m_find_last, &QPushButton::pressed, this, &find_dialog::find_last);
+
+	show();
+}
+
+find_dialog::~find_dialog()
+{
+}
+
+void find_dialog::find_next()
+{
+	if (!m_text_edit || m_find_bar->text().isEmpty())
+		return;
+
+	//m_text_edit->moveCursor(QTextCursor::Start);
```

Comment:
Leftover?
Still usually a good idea to document commented out code (mostly the reason why it's not uncommented).